packages/firestore/src/api/database.ts

```diff
@@ -489,30 +478,27 @@ export class Firestore implements firestore.FirebaseFirestore, FirebaseService {
 
   private makeDatabaseInfo(): DatabaseInfo {
     return new DatabaseInfo(
-      this._config.databaseId,
-      this._config.persistenceKey,
-      this._config.settings.host,
-      this._config.settings.ssl,
-      this._config.settings.forceLongPolling
+      this._databaseId,
+      this._persistenceKey,
+      this._settings.host,
+      this._settings.ssl,
+      this._settings.forceLongPolling
     );
   }
 
   private configureClient(
     persistenceSettings: InternalPersistenceSettings
   ): Promise<void> {
-    assert(
-      !!this._config.settings.host,
-      'FirestoreSettings.host cannot be falsey'
-    );
+    assert(!!this._settings.host, 'FirestoreSettings.host cannot be falsey');
```

Comment:
s/cannot be falsey/is not set/ if you want :)

Reply:
Heh.  SGTM.